apps/zui/src/components/icon-button.tsx

```diff
@@ -24,6 +24,7 @@ const BG = styled.button`
   justify-content: center;
   flex-shrink: 0;
   transition: background 100ms;
+  -webkit-app-region: no-drag;
```

Comment:
This was the fix.

Reply:
The app thought these buttons were draggable regions and intercepted all mouse events.